src/flownet/config_parser/_config_parser.py

```diff
@@ -1734,6 +1734,26 @@ def parse_config(
             "interpolation option for relative permeability."
         )
 
+    if (
+        config.model_parameters.equil.scheme != "regions_from_sim"
+        and config.model_parameters.equil.scheme != "individual"
+        and config.model_parameters.equil.scheme != "global"
+    ):
+        raise ValueError(
+            f"The equil scheme "
+            f"'{config.model_parameters.equil.scheme}' is not valid.\n"
+            f"Valid options are 'global', 'regions_from_sim' or 'individual'."
```

Comment:
In essence you _could_ use that list also here to generate the `ValueError` text. Not needed, just a thought.